### PR TITLE
Add delete button for events

### PR DESCRIPTION
**Pull request by Bryanroal**
Added button of eliminate
Added metod and rute
For more infomation see issue #20

### CODE REVIEW

File: app/controllers/events_controller.rb [hunk 1]
```diff
@@ -32,7 +32,13 @@ def update
       render :edit
     end
   end
+  
+  def destroy
+    @event =  Event.find(params[:id])
```
Comment (juperameza) [inline app/controllers/events_controller.rb]: you're repeating this line multiple times move it to another function and add a before action. [Read this](https://dannemanne.com/posts/ruby_on_rails_explained_before_action)

File: config/routes.rb [hunk 2]
```diff
@@ -3,6 +3,8 @@
 
   # Reveal health status on /up that returns 200 if the app boots with no exceptions, otherwise 500.
   # Can be used by load balancers and uptime monitors to verify that the app is live.
+  delete'/events/:id',  to: 'events#destroy'
```
Comment (juperameza) [inline config/routes.rb]: This also looks good but. Youre doing mannually what resources do.
[Resources](https://guides.rubyonrails.org/routing.html#singular-resources)
[ANother reading](https://launchschool.com/books/demystifying_rails/read/routes_and_resources)

**Review by juperameza — CHANGES_REQUESTED** [review]

File: app/views/events/index.html.erb [hunk 3]
```diff
@@ -25,9 +25,11 @@
...
-            </svg>
+          <%= button_to event_path(event), method: :delete do%>
+              <svg xmlns="http://www.w3.org/2000/svg" width="16" height="16" fill="currentColor" class="bi bi-trash3" viewBox="0 0 16 16">
+                <path d="M6.5 1h3a.5.5 0 0 1 .5.5v1H6v-1a.5.5 0 0 1 .5-.5M11 2.5v-1A1.5 1.5 0 0 0 9.5 0h-3A1.5 1.5 0 0 0 5 1.5v1H1.5a.5.5 0 0 0 0 1h.538l.853 10.66A2 2 0 0 0 4.885 16h6.23a2 2 0 0 0 1.994-1.84l.853-10.66h.538a.5.5 0 0 0 0-1zm1.958 1-.846 10.58a1 1 0 0 1-.997.92h-6.23a1 1 0 0 1-.997-.92L3.042 3.5zm-7.487 1a.5.5 0 0 1 .528.47l.5 8.5a.5.5 0 0 1-.998.06L5 5.03a.5.5 0 0 1 .47-.53Zm5.058 0a.5.5 0 0 1 .47.53l-.5 8.5a.5.5 0 1 1-.998-.06l.5-8.5a.5.5 0 0 1 .528-.47M8 4.5a.5.5 0 0 1 .5.5v8.5a.5.5 0 0 1-1 0V5a.5.5 0 0 1 .5-.5"/>
+              </svg>
```
Comment (MauricioNC) [inline app/views/events/index.html.erb]: You can put the svg file in a different file and just render it, by this way your code will looks more readable

**Review by MauricioNC — CHANGES_REQUESTED** [review]

**Review by juperameza — APPROVED** [review]
LGTM

**Review by MauricioNC — APPROVED** [review]

**Pull request merged**